packages/driver/cypress/e2e/e2e/origin/commands/storage.cy.ts

```diff
@@ -114,7 +114,7 @@ context('cy.origin storage', { browser: '!webkit' }, () => {
 
         expect(consoleProps.name).to.equal('clearLocalStorage')
         expect(consoleProps.type).to.equal('command')
-        expect(consoleProps.props.Yielded).to.be.null
+        expect(consoleProps.props.Yielded).to.deep.equal({})
```

Comment:
Alright....I see absolutely no reason why this Yielded should ever have been null...but this test is now failing. My only guess is that some improved performance has resulted in this change of behavior - where it wasn't set in time before, but it is now??

1. clearLocalStorage returns a Storage object (state('window').localStorage)
2. The default consoleProps() function automatically sets Yielded to the command's return value
3. When the Storage object gets serialized for cross-origin logging, it becomes {}
4. The test should expect {}, not null